src/psyclone/dynamo0p3.py

```diff
@@ -735,6 +791,15 @@ def _validate(self, need_evaluator):
                 "need an evaluator because no basis or differential basis "
                 "functions are required".format(self.name, self._eval_shape))
 
+        # Check that this kernel only updates a single argument if an
```

Comment:
Please can I postpone this to the intergrid-metadata issue because I've already done it on that branch (https://github.com/stfc/PSyclone/blob/intergrid_mdata/src/psyclone/dynamo0p3.py)?